src/go/intelligence/config.md

```diff
@@ -170,7 +174,7 @@ YOU ARE FREE TO SET VALUES FOR NEW SERVICES THOUGH.
     "url": "https://monit-grafana.cern.ch",
     "dashboardSearchAPI": "/api/search",
     "annotationAPI": "/api/annotations",
-    * "tags": ["cmsweb-play"],
+    * "tags": ["cmsweb", "das", "jobs", "prod"],
```

Comment:
please remove das since das is part of cmweb as well

Reply:
Done !